### ISSUE TITLE
Committee chairs and members should not be able to control vocabulary.

### ISSUE THREAD

**Issue by devangm** [TAMULib/Vireo#57]
- Currently, students fill these fields out, and I would prefer that remain so. Otherwise, faculty fluctuate so much that it would be onerous to update the list constantly. We would also be inundated with questions when students couldn’t find a committee member. Correcting the occasional student mistake would be easier than keeping the faculty list current.

**Comment by jmicah** [comment]
Could this be accomplished by modifying the field from the admin UI instead of in code?

**Comment by wwelling** [comment]
@jmicah yes, you should be able to configure these inputs from the admin settings.

As admin:
- select Organization tab
- select organization to modify
- select Manage workflow
- expand DOCUMENT INFORMATION
- select field to edit

![Image](https://github.com/user-attachments/assets/3518def8-de38-4f06-b062-7a6cdfe2c72f)
![Image](https://github.com/user-attachments/assets/f0b8bae2-19f0-48b7-86d6-abd789573015)
![Image](https://github.com/user-attachments/assets/0b35927a-44ad-4191-988e-6016605e606f)


I would recommend that we still back Committee chairs and members by a controlled vocabulary and allow free text if not found. I would not anticipate anyone manually maintaining this controlled vocabulary of people, but it rather be some automation tasks in the future.

Also, controlled vocabularies can be maintained with a CSV.

![Image](https://github.com/user-attachments/assets/1aa28be2-69a3-419f-92ca-4a6128281eb5)
